client/App/styles/reset.scss

```diff
@@ -0,0 +1,60 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+* {
+  font-family: Lato, system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial;
+  margin: 0;
+}
+
+html,
+body {
```

Comment:
move this part to styles

Reply:
Why? It's not part of styles resetting?

Reply:
Bc here you have the most important styles in css and you don't reset anything. i think it's little confusing.